tools/clang/test/HLSLFileCheck/validation/callgraph/barrier-group-in-nested-fn-vs-lib68.hlsl

```diff
@@ -0,0 +1,56 @@
+// RUN: %dxc -T lib_6_8 -Vd %s | %D3DReflect %s | %FileCheck %s -check-prefixes=RDAT
+
```

Comment:
Should validation be disabled here? 

I like to have explanations of the intent of a test at the top. 

```suggestion

// Verifies that use of a group barrier in a nested call in a vs library shader
// produces no diagnostics and serves as the source for like-named .ll files

```

Reply:
Yes, Validation is disabled because this HLSL is making sure the ShaderCompatInfo has the expected flags set to produce an illegal call graph which will be rejected by validation, which is used to generate .ll tests.

I'll add a comment, but the intent of the test isn't quite as stated in your suggestion, so I'll write it differently.